src/ui/flutter_app/lib/services/mill/src/engine.dart

```diff
@@ -220,3 +214,97 @@ class NativeEngine extends Engine {
     return posFenStr.toString();
   }
 }
+
+enum GameMode {
+  humanVsAi,
+  humanVsHuman,
+  aiVsAi,
+
+  /// Not Implemented
+  humanVsCloud,
```

Comment:
I once considered interacting with http://ninemensmorris.ist.tugraz.at:8080/ through HTTP requests, but I never got a response after sending an email to this university. I'm not sure if it can be called. If it is not suitable, the remaining code can be deleted later.